### ISSUE TITLE
Heartbeat might not work anymore

### ISSUE THREAD

**Issue by ebruchez** [orbeon/orbeon-forms#4455]
I have seen an error when coming back to my browser. So test this again and make sure it didn't regress.

**Comment by ebruchez** [comment]
Had this today again.

**Comment by ebruchez** [comment]
Will have to check whether this also happens with 2019.2.x.

**Comment by ebruchez** [comment]
- [ ] change session duration to be very short to reproduce locally
- [ ] fix

**Comment by ebruchez** [comment]
Locally, it might due to the fact that my session cookie was reused and apparently the session expiration in effect at the time of the creation of the cookie remains active. Clearing the cookies, and changing the session duration to 1 minute, I see the heartbeat events sent.

I am now trying on demo.orbeon.com after clearing my cookie.

**Comment by ebruchez** [comment]
Things seem ok on demo.orbeon.com too. I cleared the cookie and after about 48 minutes the `xxforms-session-heartbeat` event was sent. After 60 minutes passed, I checked that I could still work on the form.

So closing as not reproducible (due to my having old session cookies).

<img width="816" alt="Screen Shot 2020-03-30 at 12 43 48 PM" src="https://user-images.githubusercontent.com/105769/77954857-262db280-7284-11ea-9038-7534433d6c58.png">
